src/components/App.test.js

```diff
@@ -4,6 +4,7 @@ import { UnconnectedApp } from './App';
 
 jest.mock('util/firebase');
 jest.mock('images/facebook-logo.png');
+jest.mock('images/Cover1.jpg');
```

Comment:
Let's rename the image to something more descriptive.

Reply:
renamed to `hero-image.jpg`, resolved